tests-trio/test_trio_external_api.py

```diff
@@ -83,11 +83,11 @@ class ExternalAPIService(Service):
 
     class MyService(Service):
         async def run(self):
-            await self.manager.wait_finished()
+            await self._manager.wait_finished()
 
         @external_api
         async def do_scheduling(self):
-            self.manager.run_task(self.set_done)
+            self._manager.run_task(self.set_done)
 
```

Comment:
And on the 3 places above as well

Reply:
These were not intentional.  My original implementation moved it to the private variable but didn't implement the `@property` method of accessing.  Looks like I never reverted these changes.  Fixing now.